Microsoft.Toolkit.Uwp.UI.Controls/Graph/DelegateCommand.cs

```diff
@@ -0,0 +1,74 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Windows.Input;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.Graph
+{
+    /// <summary>
+    /// Represents a command that can perform a given action.
+    /// </summary>
+    public class DelegateCommand : ICommand
```

Comment:
Does this need to be public?

Reply:
@dotMorten Yes, this could be reused by other controls when need.

Reply:
It could be, but then it doesn't belong here. It should be internal until we decide to make a set of commanding APIs (this was discussed before and was then decided that's typically the job of MVVM frameworksk to provide)